PROTOCOL.grl

```diff
@@ -32,28 +32,59 @@ Where magic is defined as:
 ```
 
 The subsequent section contains a sorted list of certificate serial numbers
-associated with an index. Each entry has a fixed size in order to allow quick
-lookups. The index is used to find the corresponding bitmap of revoked grants
+associated with an offset. Each entry has a fixed size in order to allow quick
+lookups. The offset is used to find the corresponding bitmap of revoked grants
 in the next section.
 
 ```
 	uint64 serial
 	uint64 offset
 ```
 
-The last section contains a list of bitmaps. The offset of each bit in the map
-represents the position of a grant in the certificate. This means that the
-order of grants in a certificate matters.
-
-Each bitmap is represented by:
+The last section contains a list of bitmaps corresponding to serial list entries
+from the previous section. The offset of each bit in the map for a given serial
+list entry represents the position of a grant in the certificate. This means
```

Comment:
this "list" term seems out of place.Suggestion:

[...]
The offset of each bit in the map, for a given serial, represents the position of a grant in the certificate.
[...]

Reply:
Looks good to me, updated.